src/eytzinger_benchmark.zig

```diff
@@ -0,0 +1,319 @@
+const std = @import("std");
+const assert = std.debug.assert;
+const math = std.math;
+
+const binary_search = @import("./binary_search.zig").binary_search;
+const eytzinger = @import("./eytzinger.zig").eytzinger;
+const perf = @import("./benchmarks/perf.zig");
+
+const GiB = 1 << 30;
+const searches = 100_000;
+
+const kv_types = .{
+    .{.key_size = 8, .value_size = 128},
+    .{.key_size = 8, .value_size = 64},
+    .{.key_size = 16, .value_size = 16},
```

Comment:
Great, these values for secondary index composite key sizes are correct. I just double-checked against our `CompositeKey` changes.